san/sanbase_graphql_helper.py

```diff
@@ -273,9 +273,13 @@ def _format_from_date(datetime_obj_or_str):
 def _format_to_date(datetime_obj_or_str):
     if isinstance(datetime_obj_or_str, datetime.datetime):
         datetime_obj_or_str = datetime_obj_or_str.isoformat()
+    else:
+        if 'T' in datetime_obj_or_str and 'Z' in datetime_obj_or_str:
+            dt = iso8601.parse_date(datetime_obj_or_str)
```

Comment:
Google how to validate iso8601 datetime formats

Reply:
Instead of parsing datetime and treating it as date in the else block, do the opposite - try parsing date and treat as datetime in the else block
```python
date_format = '%Y-%m-%d'
try:
  date = datetime.datetime.strptime(date_str, date_format)
  # treat as date, old behaviour
except ValueError, e:
  # treat as datetime, new behavior
```